addons/mail/static/src/discuss/core/common/thread_actions.js

```diff
@@ -22,6 +22,57 @@ class ChannelActionDialog extends Component {
     `;
 }
 
+registerThreadAction("set-favorite", {
+    /**
+     * @param {Object} param0
+     * @param {import("models").DiscussChannel} param0.channel
+     */
+    condition: ({ channel, owner }) =>
+        channel &&
+        channel.self_member_id &&
+        !channel.self_member_id.is_favorite &&
+        !owner.isDiscussContent,
+    icon: "fa fa-fw fa-star",
+    name: _t("Set favorite"),
+    /**
+     * @param {Object} param0
+     * @param {import("models").DiscussChannel} param0.channel
+     * @param {import("models").Store} param0.store
+     */
+    onSelected: async ({ channel, store }) => {
+        store.fetchStoreData(
+            "/discuss/channel/favorite",
+            { channel_id: channel.id, is_favorite: true },
+            { readonly: false, silent: false }
+        );
+    },
+    sequence: 15, // after notification-settings, before advanced-settings
+    sequenceGroup: 30,
```

Comment:
Position looks off in-between "Settings" actions:
<img width="185" height="323" alt="Image" src="https://github.com/user-attachments/assets/16033fdd-fd21-483c-a5a2-532c3956d5be" />

I prefer to have it in the category above, so "Mark as Read", "Invite People", etc. but at the bottom of this category:
<img width="184" height="318" alt="Image" src="https://github.com/user-attachments/assets/99997877-730f-4191-8ccb-2dec4d2c3a01" />

IMO this feels a more natural place.

Reply:
Moved above notif